modules/subjects/src/app/store/actions/labs.actions.ts

```diff
@@ -221,7 +221,7 @@ export const cancelLabFileSuccess = createAction(
 
 export const setLabsSubGroups = createAction(
   '[Labs] Set Labs SubGroups',
-  props<{ subGroups:SubGroup[] }>()
+  props<{ subGroups }>()
```

Comment:
It is not correct to simply remove this assumption. It is necessary to update the class so that it can be used as before with the new structure or create a new class

Reply:
I don't know why, but this class is used in other parts of the code where a suitable structure is possible. If I were to modify this class, it would break the functionality in other sections where it is reused. I could create a specific class for this action, but you have prohibited me from introducing unrelated or unnecessary lines of code